### PR TITLE
added redux

### PR DESCRIPTION
**Pull request by TuyishimireEric**
## Bookstore: add reducers and actions
In this branch, I have configured Redux Store and written actions and reducer for adding and removing books. 

### Highlights

- [x]  I have added Redux Toolkit
- [x] I structured the application files using a "feature folder" approach
- [x]  I have written the book's actions and reducer (in /src/redux/books/books.js) using the ducks pattern
- [x]  I have written the categories actions and reducer (in /src/redux/categories/categories.js) using the ducks pattern
- [x] I also Configure the Redux Store (/src/redux/configureStore.js):

**Cheers and happy reviewing! :joy_cat:**

### CODE REVIEW

File: src/redux/categories/categories.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const CHECK_STATUS = 'CHECK_STATUS';
```
Comment (tufoinnkuo10) [inline src/redux/categories/categories.js]: - [x] Great job. Everything looks perfect. Please kindly ensure that The value of action type follows the [ducks pattern](https://github.com/erikras/ducks-modular-redux)

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @TuyishimireEric 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally

Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/redux/categories/categories.js (hunk 1 above)
Comment (TuyishimireEric) [reply src/redux/categories/categories.js]: Thank you @tufoinnkuo10 for reviewing my project, 
Thank you for your motivating feedback,

I have modified the value of check_status to follow the ducks pattern as suggested

THANKS

**Review by julie-ify — APPROVED** [review]
Hi @TuyishimireEric,

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by TuyishimireEric** [comment]
Thank you so much @julie-ify for approving my project :smile:

**Pull request merged**